cmd/cli/mesh_list.go

```diff
@@ -64,13 +74,26 @@ func (l *meshListCmd) run() error {
 
 	w := newTabWriter(l.out)
 
-	fmt.Fprintln(w, "\nMESH NAME\tNAMESPACE\tCONTROLLER PODS\tVERSION")
+	fmt.Fprintln(w, "\nMESH NAME\tNAMESPACE\tCONTROLLER PODS\tVERSION\tSMI SUPPORTED")
+
 	for _, elem := range list.Items {
 		m := elem.ObjectMeta.Labels["meshName"]
 		ns := elem.ObjectMeta.Namespace
 		x := getNamespacePods(l.clientSet, m, ns)
+
+		var smiList []string
+		smiMap, err := getSupportedSmiForControllerPod(x["Pods"][0], ns, l)
```

Comment:
Now that you are at it, I just checked `getNamespacePods` and you'd rather add a size check (and even `"Pods"` check). Neither is guaranteed to exist in the returned map.

Reply:
Thanks @eduser25. Added the `"Pods"` key check and length check.

![image](https://user-images.githubusercontent.com/13926417/116328462-ea626680-a77d-11eb-9e3a-cd69970b0f4b.png)